src/get-started/test-service.md

```diff
@@ -123,19 +132,19 @@ persisted by the blockchain.
 #[test]
 fn test_create_wallet() {
     let mut testkit = init_testkit();
-    let (pubkey, key) = crypto::gen_keypair();
+    let (pubkey, key) = gen_keypair();
     testkit.create_block_with_transactions(txvec![
-        TxCreateWallet::new(&pubkey, "Alice", &key),
+        TxCreateWallet::sing("Alice", &pubkey, &key),
     ]);
     let wallet = {
```

Comment:
Nit / off-topic: I wonder If additional scope is really necessary here; seems redundant.